crypto/txscript/src/opcodes/macros.rs

```diff
@@ -121,5 +121,43 @@ macro_rules! opcode_list {
                 _ => None
             }
         }
+
+        #[cfg(test)]
+        use crate::script_builder::{ScriptBuilder, ScriptBuilderResult};
+
+        #[cfg(test)]
+        #[allow(unused_comparisons)]
+        pub(crate) fn parse_short_form(test: String) -> ScriptBuilderResult<Vec<u8>>
```

Comment:
Why call it `test`?

Reply:
Why have you dropped the `version` arg? When we'll have multiple version the result of this function can be affected by the script pub key version